forms-flow-theme/scss/_modal.scss

```diff
@@ -213,4 +213,198 @@
   max-width: 100%;
   margin: auto;
   
-}
\ No newline at end of file
+}
+
+//settings modal
+
+.settings-modal{
+  display: flex;
+flex-direction: column;
+align-items: flex-start;
+align-self: stretch;
+border-radius: var(--spacer-150);
+padding-top: 5rem;
+padding-bottom: 5rem;
+
+}
+
+
+
+
+
+.section{
+display: flex;
+padding: var(--spacer-200) var(--spacer-250) !important;
+flex-direction: column;
+align-items: flex-start;
+gap: var(--spacer-100);
+align-self: stretch;
+}
+
+.modal-hr{
+border-bottom: 1px solid var(--ff-gray-000); 
+}
+
+
+.info-panel{
+display: flex;
+padding: var(--spacer-100);
+flex-direction: column;
+align-items: flex-start;
+gap: var(--spacer-050);
+align-self: stretch;
+border-radius: var( --radius-0100);
+background: var(--ff-gray-000);
+}
+
+.info-content{
+align-self: stretch;
+color: var(--ff-gray-800, #303436);
+
+
+font-size: var(--font-size-xs);
+line-height: var(--text-line-height); 
+}
+
+.url-input{
+display: flex;
+align-items: center;
+align-self: stretch;
+border-radius: var(--radius-0100);
+border: 1px solid var(--ff-base-600, #253DF4);
+}
+.settings-modal .list-group {
+  border: 1px solid var(--ff-base-600, #253DF4);
+  border-radius: var(--radius-0100);
+}
+
+.settings-modal .list-group-item {
+  border: none;
+}
+
+
+
+
+
+
+
+.field-label{
+  font-size: var(--font-size-xs);
+}
+
+
+.input-drop-edit{
+  position: absolute;
+  z-index: 100;
+  overflow: auto;
+  scrollbar-width: none;
+  border-radius: 16px;
+  border-bottom-right-radius: var(--radius-0100);
+  border-bottom-left-radius: var(--radius-0100);
+
+  border-bottom: 1px solid var(--base-base-60, #253DF4);
+
+}
+.input-drop-create{
+  position: absolute;
+  z-index: 100;
+  height: 350px;
+  overflow: auto;
+  scrollbar-width: none;
+  border-radius: 16px;
+  border-bottom-right-radius: 16px;
+  border-bottom-left-radius: 16px;
+
+  border-bottom: 1px solid var(--ff-base-600, #253DF4);
+
+}
+
+.input-drop-view{
```

Comment:
try to add common styles for different classes by ' , ' separating the class names

Reply:
Updated